pkgs/applications/networking/testssl/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  version = "2.9.5-1";
+  name = "testssl.sh-${version}";
+
+  src = fetchFromGitHub {
+    owner = "drwetter";
+    repo = "testssl.sh";
+    rev = "v${version}";
+    sha256 = "0hz6g685jwl0c0jrdca746425xpwiwc8lnlc2gigga5hkcq8qzl9";
+  };
+
+  buildInputs = with pkgs; [
+    makeWrapper
+  ];
+
+  # Replace /bin/pwd with the actual path to pwd in coreutils
+  pwdBinPath = "${lib.makeBinPath (with pkgs; [ coreutils ])}/pwd";
+  postPatch = ''
+    sed -i -e "s|/bin/pwd|${pwdBinPath}|g" testssl.sh
+  '';
+
+  installPhase = ''
+    # Make directories
+    mkdir -p $out/bin $out/etc
+
+    # Copy etc folder
+    cp -r etc/ $out/
+
+    # Copy script
+    cp testssl.sh $out/bin/testssl.sh
+
+    # Make wrapper to let the script know where the etc folder is
```

Comment:
Seeing as this is a bash script, you could avoid the wrapper indirection by simply injecting those values into the script itself.

Reply:
I'll fix this and the notes above in an amend :)